### PR TITLE
Replaces AWS EC2 with Amazon Lightsail instructions

### PR DESCRIPTION
**Pull request by mpmcroy**

### CODE REVIEW

**Comment by mpmcroy** [comment]
aws-lightsail-screenshot-1
aws-lightsail-thumbnail-1

**Comment by mpmcroy** [comment]
Screen Shot 2020-04-08 at 3 55 49 PM

File: src/server_manager/messages/master_messages.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
       }
     }
   },
-  "aws_group_assign_0": {
-    "message": "Navigate to $START_OF_LINK$Instances$END_OF_LINK$.",
-    "description": "This string appears within the server setup view as an item of a list that provides instructions to configure a security group in Amazon Web Services.",
+  "aws-lightsail-firewall-0": {
```
Comment (alalamav) [inline src/server_manager/messages/master_messages.json:36]: Please use underscores in this file.

**Comment by alalamav** [comment]
I think we should send the strings for translation once #612 and #570 are merged.

File: src/server_manager/messages/master_messages.json (hunk 1 above)
Comment (mpmcroy) [reply src/server_manager/messages/master_messages.json]: Opps. Thanks!

**Pull request merged**